sdk/search/Azure.Search.Documents/src/Indexes/FieldBuilder.cs

```diff
@@ -35,9 +33,7 @@ public class FieldBuilder
                     [typeof(bool)] = SearchFieldDataType.Boolean,
                     [typeof(DateTime)] = SearchFieldDataType.DateTimeOffset,
                     [typeof(DateTimeOffset)] = SearchFieldDataType.DateTimeOffset,
-#if EXPERIMENTAL_SPATIAL
```

Comment:
If types are still in Azure.Core.Experimental, it seems all these preproc conditions throughout the PR shouldn't be removed yet.

Reply:
I was under the impression that since this code is now being enabled and made production ready, we don't want it behind a macro anymore. The macro was to disable the code itself, and not about whether the code depends on `Azure.Core.Experimental`,
The preview version of the `Search` package can depend on `Azure.Core.Experimental` with no issues. Or, so I have been told. :)